docs/Multiple-Bundle-Stores.md

```diff
@@ -36,9 +36,18 @@ cl store add --name blob --storage-type azure_blob --url azfs://devstoreaccount1
 cl upload --store blob mkdocs.yml
 ```
 
-## Steps to set up GCP
+## Steps to set up and test with GCP
+
+First, make sure the `CODALAB_GOOGLE_APPLICATION_CREDENTIALS` env var is set on the server that runs the REST server. It should be set to a path to a credentials JSON file.
+
+```
+export CODALAB_GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json
```

Comment:
@teetone Flagging this for you -- you'll need to set up this environment variable when setting up GCP on codalab.stanford.edu.